openassessment/templates/openassessmentblock/staff_area/oa_staff_area.html

```diff
@@ -18,17 +18,18 @@ <h2 class="staff-info__title ui-staff__title">
 
                 <div class="staff-info__student ui-staff__content__section">
                     <div class="wrapper--input" class="staff-info__student__form">
-                        <form id="openassessment_student_info_form">
-                            <label for="openassessment__student_username" class="label">{% trans "Enter an individual learner's username or email" %}</label>
-                            <input id="openassessment__student_username" type="text" class="value" maxlength="255">
+                        <form class="openassessment_student_info_form">
+                            <label class="label">{% trans "Enter an individual learner's username or email" %}
+                                <input type="text" class="openassessment__student_username value" maxlength="255">
+                            </label>
                             <ul class="list list--actions">
                                 <li class="list--actions__item">
-                                    <a aria-role="button" href="" id="submit_student_username" class="action--submit"><span class="copy">{% trans "Submit" %}</span></a>
+                                    <button class="action--submit action--submit-username"><span class="copy">{% trans "Submit" %}</span></button>
```

Comment:
@andy-armstrong I don't know if it can be addressed in this PR, but the display is not ideal for the username/email field - whatever text gets entered in the field is cut off at the bottom. 
![ora_coursestafftools](https://cloud.githubusercontent.com/assets/9041399/11225620/495af986-8d49-11e5-8d12-c373a6214599.png)